packages/trader/src/Modules/SmartChartBeta/Components/LaunchModal/launch-modal.scss

```diff
@@ -0,0 +1,71 @@
+// Common styles for web modal and mobile page overlay
+.launch-modal-styles {
+    text-align: center;
+    .modal-content {
+        @include mobile {
+            height: 100%;
+            justify-content: center;
+            display: flex;
+            flex-direction: column;
+        }
+        h1 {
+            font-size: 2rem;
+            font-weight: 700;
+            line-height: 3rem;
+        }
+
+        p {
+            font-size: 16px;
+            font-weight: 400;
+            line-height: 24px;
+            text-align: center;
+            color: var(--text-general);
+        }
+        img {
+            height: 176px;
+            @include mobile {
+                height: 180px;
+                width: 200px;
+                margin-left: auto;
+                margin-right: auto;
+            }
+        }
+    }
+
+    .dc-modal-footer {
+        border-top: 2px solid var(--general-section-5);
+        padding: 1.6rem 2.4rem;
+        @include mobile {
+            margin-bottom: 38px;
+            button {
+                height: 40px;
+                width: 100%;
+            }
+        }
+    }
+}
+
+// Web modal specific styles
+.dc-modal__container_launch_modal_root {
+    @extend .launch-modal-styles;
+    min-width: 432px !important;
```

Comment:
Is there the need for `!important`. Can it be rewritten to avoid using it?

Reply:
removed